src/contour/contour.desktop

```
@@ -5,7 +5,7 @@
 [Desktop Entry]
 Type=Application
 TryExec=contour
-Exec=/usr/bin/contour
+Exec=contour
```

```suggestion
Exec=@CMAKE_INSTALL_PREFIX@/bin/contour
```
and then using a preprocessor / generator / `m4` to generate the actual .desktop file.